tests/manage/test_pvc_disruptive.py

```diff
@@ -22,38 +28,182 @@ class BaseDisruption(ManageTest):
     pvc_obj = None
     storage_type = None
 
+    def create_pvcs(self):
```

Comment:
why do we need this function? you can simply call helpers.create_multiple_pvcs() from the test

Reply:
Correct. It was unnecessary. Fixed.